src/ontology/cl-edit.owl

```diff
@@ -31101,6 +31104,38 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:36543915") Annotation(ob
 
+# Class: obo:CL_4040000 (glial restricted tripotential precursor cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10719353") obo:IAO_0000115 obo:CL_4040000 "A glial precursor cell that generates oligodendrocytes and type-1 and type-2 astrocytes. It has been shown in some mammals that this cell type may express A2B5, nestin, FGFR-1, FGFR-2, FGFR-3, PLP, and DM-20 antigens. Unlike oligodendrocyte precursor cell, it does not initially express PDGFR-alpha and can differentiate into both type-1 and type-2 astrocytes.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4040000 <https://orcid.org/0000-0002-0819-0473>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4040000 "2023-04-03T14:54:32Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10719353") oboInOwl:hasBroadSynonym obo:CL_4040000 "glial restricted precursor")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11921204") oboInOwl:hasExactSynonym obo:CL_4040000 "tripotential glial-restricted precursor cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11756508") Annotation(oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/cl#abbreviation>) oboInOwl:hasRelatedSynonym obo:CL_4040000 "GRP cell")
+AnnotationAssertion(rdfs:label obo:CL_4040000 "glial restricted tripotential precursor cell")
+SubClassOf(obo:CL_4040000 obo:CL_0000123)
+SubClassOf(obo:CL_4040000 obo:CL_0011115)
+
+# Class: obo:CL_4040001 (chorionic girdle cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27280409") Annotation(oboInOwl:hasDbXref "PMID:35328607") obo:IAO_0000115 obo:CL_4040001 "Horse-specific highly invasive trophoblast cell that invades the endometrium where it forms endometrial cups.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4040001 <https://orcid.org/0000-0002-0819-0473>)
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4040001 "2023-04-03T15:08:05Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27280409") oboInOwl:note obo:CL_4040001 "Humans, mice and rats have a highly invasive (hemochorial) placentation. Horses have a non-invasive epitheliochorial placenta, except for the transient endometrial cup.")
+AnnotationAssertion(rdfs:label obo:CL_4040001 "chorionic girdle cell")
+SubClassOf(obo:CL_4040001 obo:CL_0000351)
+SubClassOf(obo:CL_4040001 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003124))
+
+# Class: obo:CL_4040002 (enteroglial cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16336493") Annotation(oboInOwl:hasDbXref "PMID:17483847") obo:IAO_0000115 obo:CL_4040002 "Glial cell that provides support to the enteric nervous system. It is involved in enteric neurotransmission, in maintaining the integrity of the mucosal barrier of the gut and serves as a link between the nervous and immune systems of the gut.")
```

Comment:
Consider adding the following distinguishing features noted in the references:

PMID:16336493
"In the nerve strands, glial processes ensheath multiaxonal bundles,2,33 which distinguishes enteric glia from all other peripheral glia. 
...
Ultrastructurally, the most conspicuous trait of enteric glia are 10 nm filaments, which criss-cross the cell body, form axial bundles in the processes and appear to firmly anchor the cells to the ganglionic surfaces."